# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/security/moz.build
# Commit: 8705284b50d4
# Full Hash: 8705284b50d4905110df9b1fab7f00d9d77d06e6
# Author: Christoph Kerschbaumer <ckerschb@christophkerschbaumer.com>
# Date: 2019-10-22 15:53:00
# Regressor Bug: 1584993
# File Overlap Count: 1
# Description:
#   Bug 1584993: Make CSP frame-ancestors work with fission enabled. r=jkt,farre,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49147
# ==============================================================================

diff -r 141755ff04e1 -r 8705284b50d4 dom/security/moz.build
--- a/dom/security/moz.build	Tue Oct 22 08:56:13 2019 +0000
+++ b/dom/security/moz.build	Tue Oct 22 08:53:47 2019 +0000
@@ -13,6 +13,7 @@
 
 EXPORTS.mozilla.dom += [
     'CSPEvalChecker.h',
+    'DOMSecurityManager.h',
     'FramingChecker.h',
     'nsContentSecurityManager.h',
     'nsContentSecurityUtils.h',
@@ -36,6 +37,7 @@
 
 UNIFIED_SOURCES += [
     'CSPEvalChecker.cpp',
+    'DOMSecurityManager.cpp',
     'FramingChecker.cpp',
     'nsContentSecurityManager.cpp',
     'nsContentSecurityUtils.cpp',